src/components/CheckIn/checkin.scss

```
@@ -0,0 +1,184 @@
+
+$body-bg: #c1bdba;
+$form-bg: #13232f;
+$white: #ffffff;
+
+$main: #1ab188;
+$main-light: lighten($main,5%);
+$main-dark: darken($main,5%);
+
+$gray-light: #a0b3b0;
+$gray: #ddd;
+
+$thin: 300;
+$normal: 400;
+$bold: 600;
+$br: 4px;
+
+
+a {
```

You have to setup a specific selector and not setup styles for globals a html tag 
```suggestion
.you_component_global_class a {
```